assets/components/atoms/select/select-multiple.js

```diff
@@ -0,0 +1,11 @@
+/* globals $ */
+
+export default () => {
+  $('.select-multiple').each((index, element) => {
+    $(element).multipleSelect({
+      placeholder: $(element).attr('data-placeholder') ? $(element).attr('data-placeholder') : '',
```

Comment:
Holly molly 😅 `$(element).attr('data-placeholder') || ''` is enough, no need of repetition